app/models/customer.rb

```diff
@@ -1,4 +1,7 @@
 class Customer < ActiveRecord::Base
+  has_many :sales, foreign_key: :seller_id
```

Comment:
Yes, but not now. I don't want to think in delete records now. I'll add something like a flag to warn a record is deleted.